app/Helpers/GeometryHelper.php

```diff
@@ -307,6 +307,14 @@ public static function getConvexHull($geoJson)
         return $result[0]->wkt ?? null;
     }
 
+    public static function updateStatusOnEdit($sitePolygon)
+    {
+        if ($sitePolygon->status === 'approved') {
+            $sitePolygon->status = 'submitted';
+            $sitePolygon->save();
+        }
+    }
```

Comment:
I think this function would be better in `app/Models/V2/Sites/SitePolygon.php` and then name it as `edit` or something like this.
`$sitePolygon->edit()`

Reply:
right, moved to model.